functional-tests/fn_bridge_withdraw_happy.py

```diff
@@ -0,0 +1,182 @@
+import time
+
+import flexitest
+from bitcoinlib.services.bitcoind import BitcoindClient
+from strata_utils import (
+    deposit_request_transaction,
+    extract_p2tr_pubkey,
+    get_address,
+    get_balance,
+)
+from web3 import Web3
+
+from constants import (
+    DEFAULT_BLOCK_TIME_SEC,
+    DEFAULT_ROLLUP_PARAMS,
+    GWEI_TO_WEI,
+    PRECOMPILE_BRIDGEOUT_ADDRESS,
+    SATS_TO_WEI,
+    UNSPENDABLE_ADDRESS,
+)
+from entry import BasicEnvConfig
+from utils import get_bridge_pubkey, get_logger
+
+# Local constants
+# D BTC
+ROLLUP_DEPOSIT_AMOUNT = DEFAULT_ROLLUP_PARAMS["deposit_amount"]
+# Gas for the withdrawal transaction
+GAS = 22_000
+# lower bound is D BTC - fees - gas with a leeway
+STRATA_DEPOSIT_LOWER_BOUND = ((ROLLUP_DEPOSIT_AMOUNT // 1.01) * SATS_TO_WEI) - (
+    2 * GAS * GWEI_TO_WEI
+)
```

Comment:
I don't think is entirely accurate. Where does the `1.01` come from (this magic number should either be a descriptive constant or explained in the comment)?

The only place I've seen this number is in the faucet but that should not matter for the withdrawal because the deposit amount is always exactly 10 BTC.

I expect the calculation to simply be the following (but do correct me if I'm missing something):

```suggestion
# lower bound is D BTC - fees
STRATA_DEPOSIT_LOWER_BOUND = ROLLUP_DEPOSIT_AMOUNT * SATS_TO_WEI - GAS * GWEI_TO_WEI
```

Reply:
That magic 1.01 number was the old value for the deposit denomination in the test env for some reason, we changed that to just use 10 BTC.